hercules-ci-agent/hercules-ci-agent-worker/CNix.hs

```diff
@@ -95,16 +107,21 @@ withEvalState
   => Ptr (Ref NixStore)
   -> (Ptr EvalState -> ConduitT i o m r)
   -> ConduitT i o m r
-withEvalState store = bracketP (liftIO $ [C.throwBlock| EvalState* {
+withEvalState store =
+  bracketP
+    ( liftIO
+        $ [C.throwBlock| EvalState* {
       Strings searchPaths;
```

Comment:
This indentation is wrong, but can presumably be fixed manually.
Other places need it too. Look for `|]`.